operations/mimir/store-gateway.libsonnet

```diff
@@ -73,16 +75,20 @@
     $.util.readinessProbe +
     $.jaeger_mixin,
 
-  newStoreGatewayStatefulSet(name, container, with_anti_affinity=false)::
+  newStoreGatewayStatefulSet(name, container, with_anti_affinity=false, nodeAffinityMatchers=[])::
```

Comment:
I know it's unrelated to this PR but it'd be nice to rename `with_anti_affinity` to `withAntiAffinity` to match the other parameters

Reply:
Ok. Let me do in a follow up PR.

Reply:
See: https://github.com/grafana/mimir/pull/6832